spec/jobs/record_claim_event_job_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+require "geckoboard"
```

Comment:
Is this `require` necessary? If the gecko board code isn't autoloaded by the gem, then it's better practice to put the dependency closer to where it is actually required, i.e. the file where the `Claim:: GeckoboardEvent` class is defined.